core/src/accounts_hash_verifier.rs

```diff
@@ -249,36 +251,30 @@ impl AccountsHashVerifier {
         if accounts_package.expected_capitalization != lamports {
             // before we assert, run the hash calc again. This helps track down whether it could have been a failure in a race condition possibly with shrink.
             // We could add diagnostics to the hash calc here to produce a per bin cap or something to help narrow down how many pubkeys are different.
+            let calculate_accounts_hash_config = CalcAccountsHashConfig {
```

Comment:
I see we create another local variable here to pass into `calculate_accounts_hash_from_index` but in the following call we create a new instance of `CalcAccountsHashConfig` inline with the function call. Either way seems fine to me, but would be nice to keep things consistent

Reply:
I'm really just nitpicking inside of the exceptional case.

For the first one, we do:
```
let calculate_accounts_hash_config = CalcAccountsHashConfig {
    use_bg_thread_pool: false,
    ..calculate_accounts_hash_config
};
calculate_accounts_hash_from_index(
    &calculate_accounts_hash_config,
);
```
Basically, let new config = old config + 1 change. Pass in new config to next function call.

For the second one, we do:
```
.calculate_accounts_hash_from_storages(
  &CalcAccountsHashConfig {
      // now that we've failed, store off the failing contents that produced a bad capitalization
      store_detailed_debug_info_on_failure: true,
      ..calculate_accounts_hash_config
  },
```
Basically, pass old config + 1 change directly to function call

Reply:
Yeah, exactly. Or alternatively declare the `second` calculate call inline as well